docs/source/model_design/risk_factors.rst

```diff
@@ -198,68 +198,115 @@ Confounding
 
-An example:
-To illustrate this, imagine that you are interested in knowing whether smoking (our risk factor of interest) influences the risk of coronary heart disease (CHD) in men aged 18 to 64 years. You conduct a cohort study with an exposed group of male smokers in this age group from the general population, and compare their risk of CHD after a number of years to that of an unexposed group of non-smoking males in the same age group. You would expect to find a higher incidence of CHD among smokers. However, males who smoke are also likely to have a higher alcohol intake, which also increases the risk of CHD. Thus, the effect that you observe for the smoking-CHD association is **mixed**up with the effect of the association between alcohol intake and CHD. Without taking account of differences in alcohol intake between smokers and non-smokers, the magnitude of the smoking-CHD association estimated from the study may be higher than the **true** value, leading to incorrect conclusions.
+In order for a factor to be a confounder (C), it must meet the following criteria:
 
-**Confounding occurs when an estimate of the association between an exposure and a disease is mixed up with the effect of another exposure on the same disease, and the two exposures are associated.** Failure to account for confounding leads to incorrectly concluding that the effect is due to one, rather than the other variable. 
+  1. Must be a risk factor, or the surrogate of a risk factor, for the outcome (O) of interest in those who are unexposed to the exposure (E) being investigated.
+  2. Must be correlated, positively or negatively, with the exposure (E). If the study population is stratified into exposed and unexposed groups, this means that (C) has a differential distribution in the two groups.
+  3. In addition, (C) should not be an intermediate step on the causal pathway between the exposure (E) and outcome (O) being investigated (else the factor is considered an :ref:`intermediate <Intermediates>`). In other words, (C) should not be caused by the exposure (E).  
 
-In order for a factor to be a confounder, it must meet the following criteria:
-1. The factor (C ) must be associated with the exposure (E) being investigated,
-2. The factor (C) must be a risk factor for the outcome of interest in those who are not exposed to the explosure (E) being investigated,
-3. In addition, the factor (C) should not be on the causal pathway between the exposure (E) and disease (D) being investigated.
+.. todo::
+  insert link to intermediates
+
+.. image:: confounding_triangle.svg
```

Comment:
i think to illustrating that confounding is causal, there should be a solid arrow from C to E